src/test/java/core/basesyntax/service/operation/BalanceOperationHandlerTest.java

```diff
@@ -0,0 +1,57 @@
+package core.basesyntax.service.operation;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class BalanceOperationHandlerTest {
+    private static Integer ZERO_BALANCE = 0;
+    private static OperationHandler balanceOperationHandler;
+    private static FruitTransaction fruitTransactionB;
+    private static FruitTransaction fruitTransactionInvalidDataB;
+
+    @BeforeClass
+    public static void beforeClass() {
+        balanceOperationHandler = new BalanceOperationHandler();
+        fruitTransactionB = new FruitTransaction(FruitTransaction.OperationType.BALANCE,
+                "banana", 100);
+        fruitTransactionInvalidDataB = new FruitTransaction(
+                FruitTransaction.OperationType.BALANCE,
+                "banana", -1);
+    }
+
+    @Test
+    public void balanceOperation_validData_ok() {
```

Comment:
```suggestion
    public void handle_validData_ok() {
```
![image](https://user-images.githubusercontent.com/20326858/230052738-263e344d-f8b3-4a62-80a7-07d6d9d34f00.png)